lnbits/core/templates/admin/index.html

```diff
@@ -216,6 +136,21 @@
         },
         tab: 'funding',
         needsRestart: false,
+        prettyFunding: new Map([
+          ["VoidWallet", "Void Wallet"],
+          ["FakeWallet", "Fake Wallet"],
+          ["CoreLightningWallet", "Core Lightning (CLN)"],
```

Comment:
we should not use CLN anymore as far as i understood it, lets not reintroduce it here.